skorch/utils.py

```diff
@@ -483,6 +484,31 @@ def get_map_location(target_device, fallback_device='cpu'):
     return map_location
 
 
+def check_is_fitted(estimator, attributes, msg=None, all_or_any=all):
```

Comment:
We can also use the sklearn version directly:

```py
from sklearn.utils.validation import check_is_fitted as sk_check_is_fitted

def check_is_fitted(...):
    try:
        sk_check_is_fitted(...)
    except NotFittedError as e:
        raise NotInitializedError(str(e))
```

Reply:
This doesn't quite work because the error message now says "...is not **initialized** yet...". Or do you believe this is not enough to justify not re-using sklearn's `check_is_fitted`?

Reply:
I believe the scikit-learn version has a `msg` argument to change the message.